ipatests/test_integration/test_replica_promotion.py

```diff
@@ -527,3 +527,58 @@ def test_replica_install_with_existing_entry(self):
         master.run_command(arg)
 
         tasks.install_replica(master, replica)
+
+
+class TestNsslapdIgnoreTimeSkew(IntegrationTest):
+    """value for nsslapd-ignore-time-skew should remain off by default for
+    dirsrv. However while installing replica, it shuold be on. This test
+    checks that while installing replica, value for nsslapd-ignore-time-skew
+    set to on and after set to off.
+
+    https://pagure.io/freeipa/issue/7211"""
+
+    num_replicas = 2
+
+    def test_nsslapd_ignore_time_skew(self):
+        master = self.master
+        tasks.install_master(master)
+        replica = self.replicas[0]
+        client = self.replicas[1]
+
+        tasks.install_replica(master, replica)
+        log_file = '/var/log/ipareplica-install.log'
+        replica_install_log = replica.get_file_contents(log_file)
+        regex = '^.*nsslapd\-ignore\-time\-skew:\n(.*)$'
+        occ_list = re.findall(regex, replica_install_log, re.M)
```

Comment:
What's `occ`?
Also, `assert len(occ_list) == 2`

Reply:
occ = occurrence.

 List contains no of occurrence for the searched param but we are interested in first 2 only.